schemas/dns.yml

```diff
@@ -54,7 +54,7 @@
         Array of 2 letter DNS header flags.
 
         Expected values are: AA, TC, RD, RA, AD, CD, DO.
-      example: [RD, RA]
+      example: "[\"RD\", \"RA\"]"
```

Comment:
Nit: Another approach when the complex value already contains `"` would have been to wrap in single quotes instead. `'["RD", "RA"]'`. YAML will interpret the outer string and the final result will be exactly the same. Just less backslashes to write :-)

But no need to adjust, this if fine.